enumfields/fields.py

```diff
@@ -127,6 +159,10 @@ def formfield(self, form_class=None, choices_form_class=None, **kwargs):
             **kwargs
         )
 
+    def validate(self, value, model_instance):
+        self._raise_error_if_initial_value_not_allowed(value, model_instance)
+        self._raise_error_if_next_value_not_allowed(value, model_instance)
```

Comment:
Shorter and maybe better names of the methods would be:
* validate_inital_value
* validate_next_value

Reply:
Also shouldn't you call super method?

Reply:
methods renamed. Yes, calling super now. Thanks